confgenerator/testdata/invalid/linux/logging-processor_parse_multiline_only_one_language_exceptions_allowed/golden_error

```diff
@@ -0,0 +1,10 @@
+[11:7] unknown field "field"
```

Comment:
The error message here does not match the expectation. I guess we just need to remove `field: message` from the input.yaml.